docs/topics/downloader-middleware.rst

```diff
@@ -293,6 +293,52 @@ Here's an example of a log with :setting:`COOKIES_DEBUG` enabled::
     [...]
 
 
+.. _cookiejars:
+
+Direct access to cookiejars from spider
```

Comment:
Maybe this section could go before `COOKIES_ENABLED`?

Reply:
I am not sure about this. Obviously `COOKIES_ENABLED` is more important than this, then as I think it should be after as it placed now